Examples/Modules/qed/breit_wheeler/inputs_3d_optical_depth_evolution

```diff
@@ -153,6 +158,7 @@ ele_bw.momentum_distribution_type = "gaussian"
 ele_bw.xmin = 1 ## Ugly trick to avoid electrons at T=0
 ele_bw.xmax = -1 ##  Ugly trick to avoid electrons at T=0
 ele_bw.do_qed = 0
+ele_bw.plot_species = 0
```

Comment:
Actually, aren't `ele_bw` and `pos_bw` the most important species to check in this test??

Reply:
The test checks the evolution of the optical depth of a bunch of photons. For the simulation a built-in table lookup table with low resolution is used, but it still makes sense from the physical point of view. However, for the actual pair generation process, the code uses a built-in 2D lookup table which has essentially no physical meaning (too low resolution). However, as you say, it could still make sense to plot the data in order to check if something changes...

Reply:
Should I do that?